src/test/java/uk/gov/hmcts/reform/idam/web/util/TestConstants.java

```diff
@@ -156,6 +157,7 @@
     public static final String CLIENT_ID = "clientId";
     public static final String REDIRECTURI = "redirectUri";
     public static final String CUSTOM_SCOPE = "manage-roles";
+    public static final String AUTHENTICATE_SESSION_COOKE = "IPlanetDirectoryPro=A_TASTY_TREAT";
```

Comment:
COOKIE mispelt

Reply:
Its in a test class, if you make me rerun the pipeline for this I wont bring you anything back from Italy (-_-)